src/main.js

```diff
@@ -24,11 +26,29 @@ function getRandomIndex(array) {
   return array[i]
 }
 
-title.innerText = getRandomIndex(titles);
-tagline1.innerText = getRandomIndex(descriptors);
-tagline2.innerText = getRandomIndex(descriptors);
-  if (tagline1 === tagline2) {
+function randomize() {
+  title.innerText = getRandomIndex(titles);
+  tagline1.innerText = getRandomIndex(descriptors);
   tagline2.innerText = getRandomIndex(descriptors);
-  };
-  //come back to this if statement
-coverImage.src = getRandomIndex(covers)
+    if (tagline1 === tagline2) {
+      tagline2.innerText = getRandomIndex(descriptors);
+    };
+  coverImage.src = getRandomIndex(covers);
+}
+//   var randomTagline1 = getRandomIndex(descriptors);
+//   tagline1.innerText = randomTagline1;
+//
+//   var randomTagline2 = getRandomIndex(descriptors);
+//   tagline2.innerText = randomTagline2;
+//     if (tagline1 === tagline2) {
+//       randomTagline2 = getRandomIndex(descriptors);
+//       tagline2.innerText = randomTagline2;
+//     };
+//     //come back to this if statement
+//   var randomImage = getRandomIndex(covers);
+//   coverImage.src = randomImage;
+// }
+
+randomize();
+
+var currentCover = new Cover(coverImage.src, title.innerText , tagline1.innerText, tagline2.innerText);
```

Comment:
This variable probably should be on line 14 or 15, grouped with the other variables.